ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/SubmissionValidator.scala

```diff
@@ -293,17 +293,22 @@ class SubmissionValidator[LogResult] private[validator] (
 }
 
 object SubmissionValidator {
+
   type RawKeyValuePairs = Seq[(Bytes, Bytes)]
 
   type StateMap = Map[DamlStateKey, DamlStateValue]
   type LogEntryAndState = (DamlLogEntry, StateMap)
 
+  // FIXME: https://github.com/digital-asset/daml/issues/5164
+  // This should be made configurable
+  private[this] val engineConfig = Engine.DevConfig
```

Comment:
Please remove this. The `Engine` instance is already passed to the below factory method hence you only need to change the default parameter value.

Reply:
Good catch. I take care of that.